python/orca/tutorial/NCF/tf_resume_spark_dataframe.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Step 0: Import necessary libraries
+import math
+
+import tensorflow as tf
+
+from utils import *
+from process_spark_dataframe import get_feature_label_cols
+
+from bigdl.orca.learn.tf2 import Estimator
+
+
+# Step 1: Init Orca Context
+args = parse_args("TensorFlow NCF Resume Training with Spark DataFrame")
+init_orca(args, extra_python_lib="tf_model.py")
+spark = OrcaContext.get_spark_session()
+
+
+# Step 2: Read and process data using Spark DataFrame
+train_df = spark.read.parquet(os.path.join(args.data_dir,
+                                           "train_processed_dataframe.parquet"))
+test_df = spark.read.parquet(os.path.join(args.data_dir,
+                                          "test_processed_dataframe.parquet"))
+feature_cols, label_cols = get_feature_label_cols()
+
+
+# Step 3: Distributed training with Orca TF2 Estimator and load the model weight
+est = Estimator.from_keras(backend=args.backend)
+est.load(os.path.join(args.model_dir, "NCF_model"))
+
+batch_size = 10240
+train_steps = math.ceil(train_df.count() / batch_size)
+val_steps = math.ceil(test_df.count() / batch_size)
+
+callbacks = [tf.keras.callbacks.TensorBoard(log_dir=os.path.join(args.model_dir, "logs"))] \
+    if args.tensorboard else []
+
+if args.lr_scheduler:
+    lr_callback = tf.keras.callbacks.LearningRateScheduler(scheduler, verbose=1)
+    callbacks.append(lr_callback)
+
+est.fit(train_df,
+        epochs=5,
+        batch_size=batch_size,
+        feature_cols=feature_cols,
+        validation_data=test_df,
+        validation_steps=val_steps,
+        label_cols=label_cols,
+        steps_per_epoch=train_steps,
```

Comment:
reorder the arguments: steps_per_epoch, validation_data, validation_steps, feature_cols, label_cols